app/_variables.scss

```diff
@@ -3,30 +3,25 @@ $main_blue0: #6096ff; // main color
 $main_blue1: #3e7fff;
 $main_blue_light0: #81acff; // main color when hover
 $main_blue_light1: #ecf1fa;
-$main_blue_light2: #f1f6fd;
-$dark-blue0: #1b2a36;
 
-$gray0: #f8f9fb;
-$gray1: #f1f3f6;
-$gray2: #e7eaef;
-$gray3: #d8dde7;
-$gray4: #bbc2d0;
-$gray5: #9aa3b5;
-$gray6: #666d7c;
-$gray7: #f2f3f4;
```

Comment:
this color is still used at `app/components/collectionShow/collectionPaperItem.scss`